test/features/deploy_quarkus_service.feature

```diff
@@ -150,6 +150,9 @@ Feature: Deploy quarkus service
       | native   | minutes |
       | disabled | 10      |
 
+    # Disabled until Quarkus 1.6+ is used due to https://github.com/quarkusio/quarkus/issues/9954 
+    # Follow-up issue: https://issues.redhat.com/browse/KOGITO-2616
+    @disabled
```

Comment:
cherry-pick from https://github.com/kiegroup/kogito-cloud-operator/pull/418 for the tests